target/linux/ath79/modules.mk

```diff
@@ -13,4 +13,19 @@ define KernelPackage/leds-reset/description
  Kernel module for LEDs on reset lines
 endef
 
+define KernelPackage/leds-spi-byte
+  SUBMENU:=$(LEDS_MENU)
+  TITLE:=simple SPI LED controller support
+  DEPENDS:= @TARGET_ath79
```

Comment:
Do you've any reason for this dependency?

Reply:
Currently it only works on the airCube and there is no other device supporting it. But you are right: it's not really necessary. I'll remove it.

Reply:
Oh and maybe its a copy and paste one: leds-reset in the same file has it too.

Reply:
Just noted: With the change I have to move the patches to the common kernel patches and rebuild everything. Please wait with the merge till I did that.